diff -up ./Makefile.cfg.ldflags ./Makefile.cfg
--- ./Makefile.cfg.ldflags	2006-07-14 08:38:58.000000000 -0300
+++ ./Makefile.cfg	2008-07-18 05:23:24.000000000 -0300
@@ -313,7 +313,6 @@ endif
 WARN     = -Wall -Wstrict-prototypes
 INCLUDES = -I$(srcdir)/include -I.
 CFLAGS   = $(WARN) $(DLLFLAGS) $(INCLUDES) $(OPTIMIZE) $(DEFINES) 
-LDFLAGS   = -s
 
 # additional flags for shared lib.
 DLLFLAGS = -fPIC -DPIC
diff -up ./gl/Makefile.ldflags ./gl/Makefile
--- ./gl/Makefile.ldflags	1999-07-18 05:14:45.000000000 -0300
+++ ./gl/Makefile	2008-07-18 05:19:03.000000000 -0300
@@ -29,7 +29,7 @@ all:	libvgagl.a
 .PHONY: all clean dep
 
 libvgagl.so.$(VERSION): $(MODULES)
-	$(CC) -s -shared -Wl,-soname,libvgagl.so.$(MAJOR_VER) -o libvgagl.so.$(VERSION) \
+	$(CC) -shared -Wl,-soname,libvgagl.so.$(MAJOR_VER) $(LDFLAGS) -o libvgagl.so.$(VERSION) \
 	  $(MODULES)
 
 libvgagl.a: $(MODULES)
diff -up ./src/Makefile.ldflags ./src/Makefile
--- ./src/Makefile.ldflags	2006-07-14 08:30:31.000000000 -0300
+++ ./src/Makefile	2008-07-18 05:20:33.000000000 -0300
@@ -449,7 +449,7 @@ all:	libvga.so.$(VERSION)
 libvga.so.$(VERSION): $(ALLOBJS)
 	$(CC) -shared -Wl,-soname,libvga.so.$(MAJOR_VER) \
 	-Xlinker --retain-symbols-file -Xlinker ../src/exports \
-	-Wl,-Bsymbolic -o libvga.so.$(VERSION) $(ALLOBJS) -lm
+	-Wl,-Bsymbolic -o libvga.so.$(VERSION) $(ALLOBJS) -lm $(LDFLAGS)
 
 $(sharedlibdir)/libvga.so.$(VERSION): libvga.so.$(VERSION)
 	$(INSTALL_SHLIB) $< $(sharedlibdir)/$<
diff -up ./threeDKit/Makefile.ldflags ./threeDKit/Makefile
--- ./threeDKit/Makefile.ldflags	2006-07-14 08:30:49.000000000 -0300
+++ ./threeDKit/Makefile	2008-07-18 05:20:58.000000000 -0300
@@ -53,7 +53,7 @@ all:	lib3dkit.so.$(VERSION) plane wrapde
 # These rules are for ELF only.
 lib3dkit.so.$(VERSION): $(OBJECTS)
 	$(CC) -shared -Wl,-soname,lib3dkit.so.$(MAJOR_VER) \
-	  -o lib3dkit.so.$(VERSION) $(OBJECTS) -lm
+	  -o lib3dkit.so.$(VERSION) $(OBJECTS) -lm $(LDFLAGS)
 
 $(sharedlibdir)/lib3dkit.so.$(VERSION): lib3dkit.so.$(VERSION)
 	$(INSTALL_SHLIB) $< $(sharedlibdir)/$<
